### DISCUSSION TITLE
Using msw standalone (CDN) mode

### DISCUSSION THREAD

**Discussion by xujiamu** [mswjs/msw#1387] [Q&A]
Is there an example or tutorial using msw standalone?

Thanks.

**Comment by kettanaito** [comment]
Hey, @xujiamu.

You can find more info about this on the [Using with CDN](https://mswjs.io/docs/recipes/usage-with-cdn) documentation page. To summarize:

1. Import the MSW script from any npm CDN;
2. It will expose a global `MockServiceWorker` variable.
3. Access things from that variable like `const { setupWorker, rest } = MockServiceWorker`.

**Reply by xujiamu** [reply]
problem as shown
![image](https://user-images.githubusercontent.com/53904639/188260383-2f9e0bec-d2f2-4326-ba17-7864cc6014d5.png)
![image](https://user-images.githubusercontent.com/53904639/188260465-0df7bf7e-deb5-44ee-9893-c71ffa733363.png)
![image](https://user-images.githubusercontent.com/53904639/188260489-3bbfc47c-6054-4281-b81c-522acfd2a30a.png)


How to use `npx msw init < PUBLIC_DIR >`

**Reply by kettanaito** [reply]
Exactly like it’s written: 

```sh
npx msw init <DIR>
```

`npx` will run MSW CLI without install it as your dependency. And you have public directory regardless of how you consume the library.
